### PR TITLE
deps: update middleware jwt v4 ->v5

### PR DESCRIPTION
**Pull request by dongowu**
#### Description (what this PR does / why we need it):

update middleware jwt v4 ->v5

### CODE REVIEW

File: middleware/auth/jwt/jwt.go [hunk 1]
```diff
@@ -104,27 +104,23 @@ func Server(keyFunc jwt.Keyfunc, opts ...Option) middleware.Middleware {
 					tokenInfo, err = jwt.Parse(jwtToken, keyFunc)
 				}
 				if err != nil {
-					ve, ok := err.(*jwt.ValidationError)
```
Comment (Casper-Mars) [inline middleware/auth/jwt/jwt.go:107]: Why do you want to change the logic here?

File: middleware/auth/jwt/jwt.go [hunk 2]
```diff
@@ -104,27 +104,23 @@ func Server(keyFunc jwt.Keyfunc, opts ...Option) middleware.Middleware {
...
-					}
 					return nil, ErrTokenParseFail
 				}
+
 				if !tokenInfo.Valid {
 					return nil, ErrTokenInvalid
 				}
-				if tokenInfo.Method != o.signingMethod {
+
+				if _, ok := tokenInfo.Method.(*jwt.SigningMethodHMAC); !ok {
```
Comment (Casper-Mars) [inline middleware/auth/jwt/jwt.go:118]: Why doesn't it support other signature methods?

File: middleware/auth/jwt/jwt.go (hunk 1 above)
Comment (dongowu) [reply middleware/auth/jwt/jwt.go:107]: Sorry, this logic has been resolved  on my next commit

File: middleware/auth/jwt/jwt.go (hunk 2 above)
Comment (dongowu) [reply middleware/auth/jwt/jwt.go]: Sorry, I misunderstood the logic of this position and have resolved this issue on my next commit

**Pull request closed without merging**